pkg/v2/relay/relay.go

```diff
@@ -53,6 +63,19 @@ func (relay *Relay) Start() {
 		tmpl.ExecuteTemplate(w, "index.html", struct{}{})
 	})
 
+	http.HandleFunc("/metrics", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
i hate this file (as you see by the comments) but this is awesome

again with the metrics gathering i think that we could put this in metrics and have a "WriteTo(io.Writer) error" interface (something along those lines)